src/components/Header/Header.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useContext, useEffect, useRef, useState } from 'react';
+import { TodoContext } from '../../TodoContext';
+import classNames from 'classnames';
+
+type Props = {};
+
+export const Header: React.FC<Props> = () => {
+  const { todos, updateTodo, addTodo, setErrorMessage } =
+    useContext(TodoContext);
+
+  const [title, setTitle] = useState('');
+  const [inputDisabled, setInputDisabled] = useState(false);
+
+  const completedTodos = todos.filter(todo => todo.completed);
+  const activeTodos = todos.filter(todo => !todo.completed);
+
```

Comment:
DRY
Instaed of filter 4 times (here and you have same code in Footer) - move those values into context